docs/mobile/state-management.md

```diff
@@ -1,7 +1,85 @@
 ## Mobile Playbook: State management
 
-#### Persist Redux/Flux store to disk to survive shutdowns and tombstoning
+State management strategies in React Native are similar to regular React applications: Flux architecture with [Redux](http://redux.js.org/) is a sensible default choice. Alternatives like [MobX](https://github.com/mobxjs/mobx) and [Relay](https://facebook.github.io/relay/) are also interesting, but not discussed at length here.
+
+#### Persist Redux/Flux store to disk
+
+Most mobile applications will need to persist their data stores to achieve a snappy user experience and offline resilience.
+
+The simplest way to achieve persistent state and offline caching is to serialize the store object to JSON and write it to disk via the [AsyncStorage](https://facebook.github.io/react-native/docs/asyncstorage.html) API, and reset the store state from the serialized representation on app startup.
+
+You can use the [redux-storage](https://github.com/michaelcontento/redux-storage) middleware with the [redux-storage-engine-reactNativeAsyncStorage]https://github.com/michaelcontento/redux-storage-engine-reactNativeAsyncStorage backend to achieve this. For performance reasons, [debouncing](https://github.com/michaelcontento/redux-storage-decorator-debounce) the serialization is usually a good idea.
 
 #### Prepare for store schema migrations/pruning on updates
```

Comment:
This chapter feels like it should be a OSS library or a boilerplate implementation, as describing the strategy only goes so far to actually having a working implementation.